README.md

```diff
@@ -12,8 +12,8 @@ This library contains reusable code for:
 ## Requirements
 
 - Boost 1.54
-- Ceres 1.12
-- Eigen 3.2.0
+- Ceres 1.13
+- Eigen 3.3
```

Comment:
I mean, since you've already updated in the PPA then this is really just a technicality